src/app/components/table_details/table_details.html

```diff
@@ -13,6 +13,17 @@ <h6>Details</h6>
                             </dl>
                         </div>
                     </div>
+                    <div class="something" ng-if="hasData(additional_meta)">
+                         <div class="detail-group" ng-repeat="row in additional_meta">
+                            <div class="detail-body">
+                                <dl class="detail"
+                                    ng-repeat="column in row">
+                                    <dt class="detail-label" ng-repeat="(k, _) in column">{{ k }}</dt>
+                                    <dd class="detail-value" ng-repeat="(_, v) in column">{{ v }}</dd>
```

Comment:
Is `ng-repeat` needed here? There's already `row in additional_meta` and `column in row` above

Reply:
So it's a little but of a hacky way to circumvent this... https://stackoverflow.com/questions/25299436/unable-to-call-object-keys-in-angularjs

Ideally we'd have:

```html
    <dt class="detail-label">{{ Object.keys(column)[0] }}</dt>
    <dd class="detail-value">{{ Object.keys(values[0] }}</dd>
```

But apparently that's not possible. :thinking: